pages/api/admin/settings/sso.ts

```diff
@@ -0,0 +1,35 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Can't we reuse the `/api/admin/connections` instead of creating a new endpoint? We can pass a query param to filter by type. This way we can reuse the existing logic for fetching connections.

Reply:
Noted